creator-node/src/diskManager.ts

```diff
@@ -422,6 +427,22 @@ export async function sweepSubdirectoriesInFiles(
     return sweepSubdirectoriesInFiles()
 }
 
+async function _touch(path: string) {
+  return new Promise((resolve, reject) => {
```

Comment:
i think all of this code can be rewritten with awaits right? fs-extra should allow `await fs.utimes()` and `await fs.close` etc?

Reply:
sick